api/app/attendance/api.py

```diff
@@ -85,8 +113,53 @@ def post(self):
                 event_name=event.name)
         )
 
-        return attendance, 201
-
+        # Other Fields
+        registration = None
+        offer = db.session.query(Offer).filter(
+            Offer.user_id == 1).filter(Offer.event_id == event_id).first()
+
+        if not offer:
+            # Check if Guest Registration
+            registration_form = db.session.query(RegistrationForm).filter(
+                RegistrationForm.event_id == event_id).first()
+            if(registration_form):
+                registration = db.session.query(GuestRegistration).filter(
+                    GuestRegistration.user_id == user_id).filter(GuestRegistration.registration_form_id == registration_form.id).first()
+
+        else:
+            # Normal registration
+            registration = db.session.query(Registration).filter(
+                Registration.offer_id == offer.id).first()
+
+        unavilable_response = None
+        if(registration is None):
+            is_guest_registration = unavilable_response
+            has_accepted_accom_award = unavilable_response

```

Comment:
has_accepted_accom_award comes from offer though?

Reply:
Thanks for spotting this! Fixed.